allure-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/ntc/database/core/BaseDatabase.java

```diff
@@ -0,0 +1,138 @@
+package com.capgemini.ntc.database.core;
+
+import com.capgemini.ntc.database.core.base.driver.DriverManager;
+import com.capgemini.ntc.database.core.base.properties.PropertiesFileSettings;
+import com.capgemini.ntc.database.core.base.runtime.RuntimeParameters;
+import com.capgemini.ntc.test.core.BaseTest;
+import com.capgemini.ntc.test.core.BaseTestWatcher;
+import com.capgemini.ntc.test.core.ITestObserver;
+import com.capgemini.ntc.test.core.ModuleType;
+import com.capgemini.ntc.test.core.analytics.IAnalytics;
+import com.capgemini.ntc.test.core.base.environment.IEnvironmentService;
+import com.capgemini.ntc.test.core.base.properties.PropertiesSettingsModule;
+import com.capgemini.ntc.test.core.logger.BFLogger;
+import com.google.inject.Guice;
+
+import ru.yandex.qatools.allure.annotations.Attachment;
+
+abstract public class BaseDatabase implements ITestObserver {
+
+	private static DriverManager driver = null;
+
+	private final static PropertiesFileSettings propertiesFileSettings;
+	private static       IEnvironmentService    environmentService;
+	private final static IAnalytics             analytics;
+	public final static String analitycsCategoryName = "Database-Module";
+
+	static {
+		// Get analytics instance created in BaseTets
+		analytics = BaseTest.getAnalytics();
+
+		// Get and then set properties information from selenium.settings file
+		propertiesFileSettings = setPropertiesSettings();
+
+		// Read System or maven parameters
+		setRuntimeParametersDatabase();
+
+		// Read Environment variables either from environmnets.csv or any other input data.
+		setEnvironmetInstance();
+	}
+
+	public static IAnalytics getAnalytics() {
+		return BaseDatabase.analytics;
+	}
+
+	public BaseDatabase() {
+		this(getDriver());
+	}
+
+	public BaseDatabase(DriverManager driver) {
+		// Add given module to Test core Observable list
+		this.addObserver();
+
+	}
+
+	@Override
+	public void addObserver() {
+		BaseTestWatcher.addObserver(this);
+	}
+
+	@Override
+	public void onTestFailure() {
+		BFLogger.logDebug("BasePage.onTestFailure    " + this.getClass()
+				.getSimpleName());
+		makeScreenshotOnFailure();
```

Comment:
for DB module, this is the place where you can save any type of additional logs, addition to classic BFLogger

please remove `makeScreenshotOnFailure()` and `makeSourcePageOnFailure()`